src/PageLayout/PageHeader.js

```diff
@@ -1,22 +1,96 @@
 import React, { Component } from 'react';
 import './PageHeader.css';
 import notification from '../icons/notification.svg';
+import NotificationBadge from 'react-notification-badge';
+import {Effect} from 'react-notification-badge';
+import Notification from './Notification/Notification';
+
 class PageHeader extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showPopUp: false,
+            notificationClicked: false,
+            notifications: []
+        };
+
+        this.notificationClicked = this.notificationClicked.bind(this);
+        this.openPopUp = this.openPopUp.bind(this);
+        this.closePopUp = this.closePopUp.bind(this);
+    }
+    componentDidMount() {
+        //TODO: query db for notifications
+        this.setState({
+            notifications: [
+                {
+                    type: 'recurring_pickup_request',
+                    content: '-L5QoXeC_UrL5tRRED3e'
+                },
+                {
+                    type: 'recurring_pickup_confirmed',
+                    content: '-XKSIDLeC_Uksd321e'
+                },
+                {
+                    type: 'recurring_pickup_confirmed',
+                    content: '-XKSIDLeC_Uksd321e'
+                },
+                {
+                    type: 'recurring_pickup_request',
+                    content: '-XKSIDLeC_Uksd321e'
+                }
+            ]
+        });
+    }
+
+    openPopUp(){
```

Comment:
also take in the index of the notification clicked (for deciding what detail popup to show in the next step)